comp110-worksheet-6/OxoBoard.cs

```diff
@@ -10,49 +10,316 @@ public enum Mark { None, O, X };
 
 	public class OxoBoard
 	{
+        int width = 3;
+        int height = 3;
+        int inARow = 3;
+        public int count = 0;
+
+        public Mark[,] Game_Board;
 		// Constructor. Perform any necessary data initialisation here.
 		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+		public OxoBoard()
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            Game_Board = new Mark[width, height];
+
+            for (int x = 0; x < width; x++)
+            {for (int y = 0; y < height; y++)
```

Comment:
The formatting of your code (indentation and placement of braces) is inconsistent in places -- it's worth adopting a consistent style for this. NB Visual Studio can automatically format your code for you -- I recommend this.